.github/vale/styles/Aiven/capitalization_headings.yml

```diff
@@ -81,6 +81,8 @@ exceptions:
   - Rivery
   - reStructuredText
   - Setting up SAML with OneLogin
+  - Set up SAML with Auth0
```

Comment:
Shouldn't be necessary to include all the phrase but only the needed words like `SAML` and `Auth0`

Reply:
Thanks @ftisiot 

I removed the sentences and only included the words via https://github.com/aiven/devportal/pull/1419/commits/484384092f9874237c8d3fa7cb5ff1602e37a131